oml/functional/metrics.py

```diff
@@ -209,6 +214,14 @@ def calculate_accuracy_on_triplets(embeddings: torch.Tensor, reduce_mean: bool =
         return acc
 
 
+def calc_fnmr_at_fmr(distances: torch.Tensor, mask_gt: torch.Tensor, fmr_val: float) -> torch.Tensor:
```

Comment:
Please, add the doc string explaining the idea of the metric and how we calculate it.
You may add an example with exact numbers from tests.
It would be also good to provide a link to the metric description. (Please, check other docs to see how we format links)

Reply:
I was trying to mimic other functions in that module. I'll add the docstring.

Reply:
thank you